kinto/core/storage/postgresql/client.py

```diff
@@ -81,6 +81,7 @@ def create_from_config(config, prefix='', with_transaction=True):
     settings.pop(prefix + 'max_size_bytes', None)
     settings.pop(prefix + 'prefix', None)
     settings.pop(prefix + 'strict_json', None)
+    settings.pop(prefix + 'hosts', None)
```

Comment:
Now that I see this, maybe a whitelist of supported settings would make more sense...

Reply:
Is there something else i need to do?

Reply:
No, don't worry we can open another issue and handle that later :)

Reply:
I created https://github.com/Kinto/kinto/issues/1410 to handle that.

Reply:
Hi @Natim, 
Can I also look into this issue (1410) ?

Reply:
Yes sure be you guest